libs/features/georoutingv2/georoutingv2.css

```diff
@@ -168,9 +151,3 @@
     max-width: 720px;
   }
 }
-
-@media (min-width: 1200px) {
-  .dialog-modal.locale-modal-v2 {
-    background-image: url('/libs/features/georoutingv2/img/GeoModal_BG_Map_Desktop.png');
```

Comment:
we need to remove the image from assets

Reply:
@narcis-radu I have removed this image from repo